### PR TITLE
TAJO-1990: Refine some parts in HBaseTablespace.

### PR DESCRIPTION
**Pull request by hyunsik**
This is a trivial patch to refine some parts in HBaseTablespace.

### CODE REVIEW

**Comment by hyunsik** [comment]
Rebased.

**Comment by hyunsik** [comment]
Fixed the test failure and rebased.

File: tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseTablespace.java [hunk 1]
```diff
@@ -702,25 +705,22 @@ public String toString() {
...
-          indexPredication.setColumn(indexableColumns[0]);
-          indexPredication.setColumnId(tableDesc.getLogicalSchema().getColumnId(indexableColumns[0].getQualifiedName()));
-          indexPredication.setStartValue(indexPredicationValues.getFirst());
-          indexPredication.setStopValue(indexPredicationValues.getSecond());
-
-          indexPredications.add(indexPredication);
-        }
-      }
+    final Column[] indexableColumns = getIndexableColumns(tableDesc);
+    if (indexableColumns == null && indexableColumns.length == 0) {
```
Comment (jihoonson) [inline tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseTablespace.java]: ```&&``` needs to be ```||```.

**Comment by jihoonson** [comment]
+1 this patch looks good to me.
Please fix the above condition before commit.

File: tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseTablespace.java (hunk 1 above)
Comment (hyunsik) [inline tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseTablespace.java]: Thank you for the comment. I'll fix it and then commit it.

**Comment by jinossy** [comment]
@hyunsik 
Could you check this error?
```
2015-12-14 15:55:20,180 FATAL org.apache.tajo.master.TajoMaster: Error starting TajoMaster
java.lang.NoClassDefFoundError: org/apache/hadoop/hbase/HConstants
        at org.apache.tajo.storage.hbase.HBaseTablespace.<clinit>(HBaseTablespace.java:76)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:264)
        at org.apache.tajo.storage.TablespaceManager.extractStorage(TablespaceManager.java:185)
        at org.apache.tajo.storage.TablespaceManager.loadStorages(TablespaceManager.java:166)
        at org.apache.tajo.storage.TablespaceManager.applyConfig(TablespaceManager.java:154)
        at org.apache.tajo.storage.TablespaceManager.initForDefaultConfig(TablespaceManager.java:120)
        at org.apache.tajo.storage.TablespaceManager.<init>(TablespaceManager.java:93)
        at org.apache.tajo.storage.TablespaceManager.<clinit>(TablespaceManager.java:84)
        at org.apache.tajo.master.TajoMaster.serviceInit(TajoMaster.java:183)
        at org.apache.hadoop.service.AbstractService.init(AbstractService.java:163)
        at org.apache.tajo.master.TajoMaster.main(TajoMaster.java:597)
Caused by: java.lang.ClassNotFoundException: org.apache.hadoop.hbase.HConstants
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 12 more
```

**Comment by hyunsik** [comment]
Fixed that. Thank you.

**Pull request closed without merging**